### PR TITLE
Interactive map of coronavirus 2019-nCov global distribution

### PR DESCRIPTION
**Pull request by darwinanddavis**
Thanks for contributing your submission to the e-Rum2020 CovidR contest.

**For CovidR contributors:**

- [x] The title of this PR reflects your contribution
- [x] The pull request only includes one new `.yml` file under `contest/contributions`
- [x] Make sure "Allow edits from maintainers" is checked
- [x] Send an email to covidrcontest@erum.io (for further direct communication down the road)
     - Subject: ~~"Submission to CovidR: PR #00", where #00 is the number of this Pull Request~~
     - Content: A full link to this Pull Request

---

For CovidR maintainers:

- [x] Label the PR as "submission"
- [x] Check the validity of the submission
- [x] Assess completeness of metadata and possibly adjust the `.yml` file-name
- [x] Add a field with the current `date` (e.g. `2020-04-01` for April 1st)
- [x] Check the integration of the new page in the website gallery (locally or via Actions' artifact)
- [x] Make sure the email to covidrcontest@erum.io was received
- [x] E-mail received
- [x] Merge the pull request
- [x] The initial comment should be automatically created, otherwise comment manually with the raw content of [vote-comment.md](../tree/master/contest/_tools/vote-comment.md) and an initial like
- [x] Provide here the link to the contribution as published to the gallery: https://milano-r.github.io/erum2020-covidr-contest/malishev-covid19.html
- [x] Instruct contributors on how to include their badge from the gallery page.

### CODE REVIEW

**Comment by riccardoporreca** [comment]
@darwinanddavis, can you please make sure the actual source code behind your contributions is available in the `repository` specified in the YAML file?

File: contest/contributions/malishev-covid19.yml [hunk 1]
```diff
@@ -5,6 +5,6 @@ abstract: |
   Interactive map of coronavirus 2019-nCov global distribution using live webscraped data from the [European Centre for Disease Prevention and Control (ECDC)](https://www.ecdc.europa.eu/en/geographical-distribution-2019-ncov-cases).  
   Data are for the country level showing cases, deaths, and cases in last 15 days, and show individual country rankings for these metrics.   
 menu_entry: Interactive map of 2019-nCov global distribution
-repository: https://github.com/darwinanddavis/worldmaps
+repository: https://github.com/darwinanddavis/worldmaps/blob/gh-pages/docs/coronavirus_.R
```
Comment (darwinanddavis) [inline contest/contributions/malishev-covid19.yml:8]: This is the R file for the map.

File: contest/contributions/malishev-covid19.yml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# Contribution's title, author, abstract
+title: Realtime interactive global distribution of 2019-nCov showing cases deaths and cases in last 15 days
```
Comment (riccardoporreca) [inline contest/contributions/malishev-covid19.yml]: Align title style (mainly title-case)
```suggestion
title: Real-time Interactive Global Distribution of 2019-nCov: Cases, Deaths and Cases in Last 15 Days 
```

**Review by riccardoporreca — CHANGES_REQUESTED** [review]
@darwinanddavis, I have suggested a small change to the title, mainly to align it to the title style / case. You can directly commit the suggestion or refine it yourself along those lines. Otherwise we are good to go!

File: contest/contributions/malishev-covid19.yml [hunk 3]
```diff
@@ -0,0 +1,10 @@
+# Contribution's title, author, abstract
+title: "Real-time Interactive Global Distribution of 2019-nCov: Cases, Deaths, and Cases in Last 15 Days" 
+author: Matt Malishev
+abstract: |
+  Interactive map of coronavirus 2019-nCov global distribution using live webscraped data from the [European Centre for Disease Prevention and Control (ECDC)](https://www.ecdc.europa.eu/en/geographical-distribution-2019-ncov-cases).  
+  Data are for the country level showing cases, deaths, and cases in last 15 days, and show individual country rankings for these metrics.   
+menu_entry: Interactive map of 2019-nCov global distribution
+repository: https://github.com/darwinanddavis/worldmaps/blob/gh-pages/docs/coronavirus_.R
+type: webpage
+content: https://darwinanddavis.github.io/worldmaps/coronavirus.html # URL of the webpage
```
Comment (riccardoporreca) [inline contest/contributions/malishev-covid19.yml]:
```suggestion
content: https://darwinanddavis.github.io/worldmaps/coronavirus.html # URL of the webpage
date: 2020-05-04-16-00
```

**Review by riccardoporreca — APPROVED** [review]
Added `date` field, ready to merge

**Comment by riccardoporreca** [comment]
@darwinanddavis, your submission is now accepted and visible at https://milano-r.github.io/erum2020-covidr-contest/malishev-covid19.html :tada: 
You can click on the **eRum2020::Covidr badge** in the page to obtain the badge code you can embed in your repository.

Furthermore, feel free to spread your contribution on social media with the link above, using hashtags `#erum2020` `#covidr`, and invite the community to like the contribution using the corresponding button.

**Pull request merged**